libs/router/generate-route-path-builder.function.ts

```diff
@@ -0,0 +1,21 @@
+import type { ISlashOptions } from '@umbraco-cms/internal/router';
```

Comment:
This type will be moved to "@umbraco-cms/backoffice/router" (so can be imported locally) after #642